source/model/Activator.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC, abstractmethod
+
+
+class Activator(ABC):
+
+    def __init__(self, id):
+        self.activatorId = id
+
+    @property
+    @abstractmethod
+    def name(self):
```

Comment:
This is a side effect from using properties. You do not get the name you just have Activator.name .
We did not get the property decorator to work 100% though. With correct functioning it should result in that you can just assign a new value to Activator.name.

Create issue #18.